tests/storage/test_event_federation.py

```diff
@@ -871,7 +869,44 @@ def test_get_backfill_points_in_room_attempted_event_retry_after_backoff_duratio
             self.store.get_backfill_points_in_room(room_id, depth_map["A"], limit=100)
         )
         backfill_event_ids = [backfill_point[0] for backfill_point in backfill_points]
-        self.assertListEqual(backfill_event_ids, ["b3", "b2", "b1"])
+        self.assertEqual(backfill_event_ids, ["b3", "b2", "b1"])
+
+    def test_repeated_get_backfill_points_in_room_succeed(self) -> None:
+        """
+        A test that reproduces #13929 (Postgres only).
```

Comment:
Is this really only Postgres only? Does SQLite support bigger numbers?

Reply:
```
sqlite> select (1<<20);
(1<<20)
1048576
sqlite> select (1<<20) * (1<<20);
(1<<20) * (1<<20)
1099511627776
```

I'd guess sqlite has a single integer type, and I'd guess that it's a 64 bit int at that?

Hmm. [The docs say:](https://www.sqlite.org/datatype3.html#storage_classes_and_datatypes)

> INTEGER. The value is a signed integer, stored in 0, 1, 2, 3, 4, 6, or 8 bytes depending on the magnitude of the value.

So I guess conceptually SQLite's ints are 64 bit, but they might get stored more compactly if they're small? It also looks like overly large integers get stored as doubles (`REAL`s):

```
sqlite> select (1<<62) * 100;
(1<<62) * 100
4.61168601842739e+20
```

Reply:
But sorry, yeah, it's Postgres only. I tried running my origin test on SQLite locally and it passed just fine.